app/assets/javascripts/index.js

```
@@ -410,4 +436,16 @@ function initFilterButtons() {
     init();
 }
 
-export { initFilterButtons, initFilterIndex, setBaseUrl };
+
+function toggleIndexReload() {
+    const loader = window.dodona.index.periodicReload;
+    if (loader !== null) {
```

This shouldn't happen right? The `toggleIndexReload` function is (should) only be used on pages where `refreshElement` wasn't `null`, which means that `window.dodona.index.periodicReload` will have been set by `initRefresh()`.

I'm not necessarily saying it should be changed, justed wanted to check.

---

Although, if someone forgets to pass `refreshElement` in the future, this will probably make it harder to debug why things aren't working. Maybe we should just let it error out in that case?

---

Correct on both accounts, I've changed it (and moved the toggle code to the class instead).